css/parser/src/parser/style_rule.rs

```diff
@@ -16,15 +16,20 @@ where
 
         loop {
             // TODO: remove `}`
-            if self.input.is_eof()? || is!(self, "}") {
+            if is!(self, "}") {
                 return Ok(rules);
             }
 
-            match cur!(self) {
-                tok!(" ") => {
-                    self.input.skip_ws()?;
+            let cur = self.input.cur()?;
```

Comment:
Code seems identical, is there a reason for this change?

Reply:
To be honestly it is the same, we need refactor logic around spaces, because now we lose them between `comments`, but let's postpone it for other PRs, I will return original

Reply:
Did you push your changess?

Reply:
Yep

Reply:
Github diff view still shows changes.
Hmm...

Reply:
Oh, we need it due we should handle `None` in `match`, css parser and lexer in spec has two definitions:
1. consume (cur + bump)
2. reconsume (reset state)

But in most of places we do not respect it, because of this, we often need to pass arguments or store intermediate (i.e. span/cur token/etc) states somewhere else, it reduces performance and requires us have more code then we need. Ideally all in lexer and parser should use `loop + cur + bump + match` and `reset` if we need reconsume, here the first improvement, after lexer refactor it will be easy to convert it everywhere.

But I can avoid it here and do it late, I think it will be even better